### PR TITLE
docs: add user attributes reference

### PR DESCRIPTION
**Pull request by owlas**
Closes: #6246

### CODE REVIEW

File: docs/docs/references/user-attributes.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      sql_filter: ${TABLE}.sales_region = {{ lightdash.attributes.sales_region }}
+```
+
+### 2. Filtering joins with `sql_on`
+
+If you're joining a table, you can also customise the rows that are returned 
+
+You can use user attributes to filter the rows returned by a join. This is useful if you want to restrict the data 
+returned from the joined table. To reference a user attribute in your sql, use the special lightdash reference 
+`{{ lightdash.attributes.<attribute_name> }}`. For example, if you have a user attribute called `sales_region` you can
```
Comment (rephus) [inline docs/docs/references/user-attributes.mdx:67]: Is this the right reference, it should be ${} not {{}}

File: docs/docs/references/user-attributes.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+User attributes can be assigned to users by navigating to **Organization Settings** > **User Attributes** and 
+clicking on the user attribute you'd like to assign. Select a user by email address and set their value.
+
+## Using user attributes in Lightdash
+
+There are several places in Lightdash where you can customise behaviour based on user attributes.
+
+### 1. Row filtering with `sql_filter`
+
+You can use user attributes to filter the rows returned by a query. This is useful if you want to restrict the data
```
Comment (rephus) [inline docs/docs/references/user-attributes.mdx:49]: should we mention different alliases ? like sql_where or `ld` `attr`

**Review by rephus — COMMENTED** [review]
Really nice docs, and amazing videos. well done

Just a few minor comments

File: docs/docs/references/user-attributes.mdx (hunk 1 above)
Comment (owlas) [reply docs/docs/references/user-attributes.mdx:67]: Thanks that's an error!

**Pull request merged**